apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -105,6 +105,12 @@ type MachineService interface {
 	// exists.
 	// It returns a NotFound if the given machine doesn't exist.
 	SetKeepInstance(ctx context.Context, machineName coremachine.Name, keep bool) error
+	// GetMachineUUID returns the UUID of a machine identified by its name.
+	// It returns a MachineNotFound if the machine does not exist.
```

Comment:
**todo:** Please check this one and remove or fix the comment about error handling, as long other occurences:


```txt
Matches for '// It returns a MachineNotFound if the machine does not exist.' in [Recent Files, Types, Files, Symbols, Endpoints, Database, Config keys, Text]  (21 usages found)
    Usage in comments  (21 usages found)
        juju  (21 usages found)
            apiserver/facades/agent/instancemutater  (1 usage found)
                instancemutater.go  (1 usage found)
                    39 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/agent/provisioner  (1 usage found)
                service.go  (1 usage found)
                    71 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/agent/storageprovisioner  (1 usage found)
                service.go  (1 usage found)
                    39 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/application  (1 usage found)
                service.go  (1 usage found)
                    54 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/charms  (1 usage found)
                service.go  (1 usage found)
                    36 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/client  (1 usage found)
                service.go  (1 usage found)
                    40 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/controller  (1 usage found)
                controller.go  (1 usage found)
                    89 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/machinemanager  (1 usage found)
                machinemanager.go  (1 usage found)
                    109 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/client/modelmanager  (1 usage found)
                services.go  (1 usage found)
                    182 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/controller/caasapplicationprovisioner  (1 usage found)
                service.go  (1 usage found)
                    60 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/controller/firewaller  (1 usage found)
                interface.go  (1 usage found)
                    56 // It returns a MachineNotFound if the machine does not exist.
            apiserver/facades/controller/instancepoller  (1 usage found)
                service.go  (1 usage found)
                    37 // It returns a MachineNotFound if the machine does not exist.
            domain/credential/service  (1 usage found)
                modelcredential.go  (1 usage found)
                    32 // It returns a MachineNotFound if the machine does not exist.
            domain/machine/service  (4 usages found)
                service.go  (4 usages found)
                    122 // It returns a MachineNotFound if the machine does not exist.
                    138 // It returns a MachineNotFound if the machine does not exist.
                    353 // It returns a MachineNotFound if the machine does not exist.
                    387 // It returns a MachineNotFound if the machine does not exist.
            domain/machine/state  (2 usages found)
                state.go  (2 usages found)
                    608 // It returns a MachineNotFound if the machine does not exist.
                    749 // It returns a MachineNotFound if the machine does not exist.
            internal/worker/bootstrap  (1 usage found)
                service.go  (1 usage found)
                    67 // It returns a MachineNotFound if the machine does not exist.
            internal/worker/computeprovisioner  (1 usage found)
                manifold.go  (1 usage found)
                    32 // It returns a MachineNotFound if the machine does not exist.
```

Reply:
Yes bad copy/paste, fixed.